agent/engine/task_manager.go

```diff
@@ -397,24 +395,26 @@ func (mtask *managedTask) handleDesiredStatusChange(desiredStatus apitaskstatus.
 func (mtask *managedTask) handleContainerChange(containerChange dockerContainerChange) {
 	// locate the container
 	container := containerChange.container
+	runtimeID := container.GetRuntimeID()
+	event := containerChange.event
+	seelog.Infof("Managed task [%s]: Container [name=%s runtimeID=%s]: handling container change event [%s]",
+		mtask.Arn, container.Name, runtimeID, event.Status.String())
+	seelog.Debugf("Managed task [%s]: Container [name=%s runtimeID=%s]: handling container change event [%s]",
```

Comment:
this looks confusing on debug level because they look like duplicate. Can the second one be somewhat different (e.g. "Managed task [%s]: Container [name=%s runtimeID=%s]: container change event: [%s]")

Reply:
sure, will do 👍